src/locales/es/arena-tag.json

```
@@ -1 +1,51 @@
-{}
\ No newline at end of file
+{
+  "yourTeam": "tu equipo",
+  "opposingTeam": "el equipo rival",
+  "arenaOnRemove": "Los efectos de {{moveName}} desaparecieron.",
+  "arenaOnRemovePlayer": "Los efectos de {{moveName}}\ndesaparecieron en tu bando.",
+  "arenaOnRemoveEnemy": "Los efectos de {{moveName}}\ndesaparecieron en el bando rival.",
+  "mistOnAdd": "¡Neblina de {{pokemonNameWithAffix}}\nha cubierto a su equipo!",
+  "mistApply": "¡La neblina evita los cambios de estadísticas!",
+  "reflectOnAdd": "¡Reflejo redujo el daño físico!",
+  "reflectOnAddPlayer": "¡Reflejo redujo el daño físico en tu bando!",
+  "reflectOnAddEnemy": "Reflejo redujo el daño físico en el bando rival.",
+  "lightScreenOnAdd": "¡Pantalla de Luz redujo el daño físico!",
+  "lightScreenOnAddPlayer": "¡Pantalla de Luz redujo el daño físico en tu bando!",
+  "lightScreenOnAddEnemy": "¡Pantalla de Luz redujo el daño físico en el bando enemigo!",
+  "auroraVeilOnAdd": "¡Velo Aurora redujo el daño físico!",
+  "auroraVeilOnAddPlayer": "¡Velo Aurora redujo el daño físico en tu bando!",
+  "auroraVeilOnAddEnemy": "¡Velo Aurora redujo el daño físico en el bando rival!",
+  "conditionalProtectOnAdd": "¡{{moveName}} protege a su bando!",
+  "conditionalProtectOnAddPlayer": "¡{{moveName}} protege a tu bando!",
+  "conditionalProtectOnAddEnemy": "¡{{moveName}} protege al bando rival!",
+  "conditionalProtectApply": "¡{{pokemonNameWithAffix}} ha sido protegido por {{moveName}}!",
+  "matBlockOnAdd": "¡{{pokemonNameWithAffix}} va a usar un tatami para bloquear ataques!",
+  "noCritOnAddPlayer": "¡{{moveName}} protege a tu bando de golpes críticos!",
+  "noCritOnAddEnemy": "¡{{moveName}} protege al bando rival de golpes críticos!",
+  "noCritOnRemove": "¡Los efectos de {{moveName}} de {{pokemonNameWithAffix}} se han disipado!",
+  "wishTagOnAdd": "¡El deseo de {{pokemonNameWithAffix}} se ha hecho realidad!",
+  "mudSportOnAdd": "¡Se han debilitado los ataques de tipo Eléctrico!",
+  "mudSportOnRemove": "Chapoteo Lodo ha dejado de surtir efecto.",
+  "waterSportOnAdd": "¡Se han debilitado los ataques\nde tipo Fuego!",
+  "waterSportOnRemove": "Hidrochorro ha dejado de surtir efecto.",
+  "spikesOnAdd": "¡El equipo de {{opponentDesc}} ha sido rodeado por {{moveName}}!",
+  "spikesActivateTrap": "¡Las púas han herido a {{pokemonNameWithAffix}}!",
+  "toxicSpikesOnAdd": "¡El equipo de {{opponentDesc}} ha sido rodeado por {{moveName}}!",
+  "toxicSpikesActivateTrapPoison": "¡{{pokemonNameWithAffix}} ha sido herido por {{moveName}}!",
+  "stealthRockOnAdd": "¡El equipo de {{opponentDesc}} ha sido rodeado por piedras puntiagudas!",
+  "stealthRockActivateTrap": "¡Unas piedras puntiagudas han dañado a {{pokemonNameWithAffix}}!",
+  "stickyWebOnAdd": "¡Una {{moveName}} se extiende a los pies del bando rival!\t",
```

```suggestion
  "stickyWebOnAdd": "¡Una {{moveName}} se extiende a los pies del bando rival!",
```

Unsure of how that "\t" got there or what it does, so I'm removing it.